src/service/url-replacements-impl.js

```diff
@@ -313,6 +313,20 @@ export class GlobalVariableSource extends VariableSource {
       return Date.now();
     });
 
+    //Returns the human readable timestamp in format of 2011-01-01 11:11:11.
```

Comment:
@cramforce that was my original recommendation, but requester wants time in local timezone. see conversation in #9959